CAIPs/caip-n.md

```diff
@@ -0,0 +1,84 @@
+---
+caip: <to be assigned>
+title: Chain-Agnostic JSON-RPC Interface
+author: Dan Finlay <dan@metamask.io>
+discussions-to: TBA
```

Comment:
can you open a thread about it on eth-magicians and add this link here?

Reply:
Funny to open chain-agnostic discussion on eth-magicians, but ok :)